packages/@coorpacademy-app-player/src/middlewares/test/index.js

```diff
@@ -2,6 +2,12 @@ import test from 'ava';
 import {createStore} from 'redux';
 import createMiddleware from '../index';
 
+const appOptions = {
+  services: {
+    Logger: {}
```

Comment:
pas de fonction `error` dans ton logger ?
Je conseillerai: `Logger: {error() { throw new Error('error logger should not have been called'); } }`